src/Catalog/Registration/RegistrationMakerCatalogItem.cs

```diff
@@ -106,7 +106,7 @@ DateTime GetPublishedDate()
                 }
             }
 
-            _listed = (_publishedDate.Year == 1900) ? false : true;
+            _listed = (_publishedDate.ToUniversalTime().Year == 1900) ? false : true;
```

Comment:
> Would a safer check be `_publishedDate.ToUniversalTime().Year <= 1900`?

This does not match the checks in other places in the code. The logic always has to do with matching against year 1900.